testing/databaseTest.py

```diff
@@ -443,14 +443,14 @@ def kinetics_checkLibraryRatesAreReasonable(self, library):
             if k < 0:
                 boo = True
                 logging.error('library reaction {0} from library {1}, had a negative rate at 1000 K, 1 bar'.format(rxn,library.label))
-            if len(rxn.reactants) == 1:
+            if len(rxn.reactants) == 1 and rxn.allow_max_rate_violation==False:
```

Comment:
The test fails with `AttributeError: 'rmgpy.reaction.Reaction' object has no attribute 'allow_max_rate_violation'`, I think you should first check `hasattr`

Reply:
Sorry, I forgot to actually set the attribute Reaction.__init__